web-console/src/views/datasource-view/datasource-view.tsx

```diff
@@ -149,15 +149,15 @@ export interface DatasourcesViewState {
   datasourcesError?: string;
   datasourceFilter: Filter[];
 
-  showDisabled: boolean;
+  showUnused: boolean;
   retentionDialogOpenOn?: RetentionDialogOpenOn;
   compactionDialogOpenOn?: CompactionDialogOpenOn;
-  dropDataDatasource?: string;
-  enableDatasource?: string;
+  dataSourceToMarkAsUnusedAllSegmentsIn?: string;
+  dataSourceToMarkAllNonOvershadowedSegmentsAsUsedIn?: string;
   killDatasource?: string;
-  dropReloadDatasource?: string;
-  dropReloadAction: 'drop' | 'reload';
-  dropReloadInterval: string;
+  dataSourceToMarkSegmentsByIntervalIn?: string;
```

Comment:
This is a great rename, it is good to have the UI aligned with the docs. Could you please make sure that we treat "datasource" as one word I have been consistently using "datasource" for the Druid table thing and "data source" for the source of data like S3 or HDFS (in the data loader).

Reply:
So this should be `datasourceToMarkSegmentsByIntervalIn`

Reply:
Renamed